modules/platforms/cpp/odbc/include/ignite/odbc/message.h

```diff
@@ -49,6 +49,17 @@ namespace ignite
             };
         };
 
+        /**
+         * Default nested tx mode for compatibility.
+         */
+        struct NestedTxType
+        {
+            enum Type
+            {
+                NESTED_TX_MODE = 3
```

Comment:
This constant has name `DEFAULT_NESTED_TX_MODE` in java code.
Let's make it consistent here.

Reply:
Fixed it